hack/test-cmd.sh

```diff
@@ -1087,13 +1087,11 @@ __EOF__
   kube::test::get_object_assert deployment "{{range.items}}{{$deployment_image_field}}:{{end}}" "${IMAGE_NGINX}:${IMAGE_NGINX}:"
   kube::test::if_has_string "${output_message}" "Object 'Kind' is missing"
   ## Pause the deployment
-  output_message=$(! kubectl rollout pause -f hack/testdata/recursive/deployment --recursive 2>&1 "${kube_flags[@]}")
+  kubectl-with-retry rollout pause -f hack/testdata/recursive/deployment --recursive 2>&1 "${kube_flags[@]}"
   kube::test::get_object_assert deployment "{{range.items}}{{.spec.paused}}:{{end}}" "true:true:"
-  kube::test::if_has_string "${output_message}" "Object 'Kind' is missing"
```

Comment:
We do, but I removed it as the only solution to triage the flake was to attempt using `kubectl-with-retry` instead of `kubectl`.

However, in doing so, `kubectl-with-retry` directs stderr to a file that is removed once it is done executing, and does not allow me to review it for this missing object as I was doing